client/ts/src/client.ts

```diff
@@ -1464,6 +1465,49 @@ export class ManifestClient {
     );
   }
 
+  /**
+   * CancelAllOnCore instruction. Cancels all orders on a market directly on the core program,
+   * including reverse orders and global orders with rent prepayment.
+   *
+   * @returns TransactionInstruction[]
+   */
+  public async cancelAllOnCoreIx(): Promise<TransactionInstruction[]> {
+    if (!this.payer) {
+      throw new Error('Read only');
+    }
+
+    const openOrders: RestingOrder[] = this.market.openOrders();
+    const cancelInstructions: TransactionInstruction[] = [];
+
+    for (const openOrder of openOrders) {
```

Comment:
there is going to be a lot of ix overhead here if there are multiple.
batch together all the cancels into the same ix

Reply:
added 25 as the limit but lmk if there should be none? I figured this will fail if there are too many in 1 ix and didn't know it the same holds true for the tx

Reply:
Assuming that I am correct on the limiter being bytes in the tx.
There is even more overhead if they are in separate ix, both bytes and CU.

Reply:
25 is a good limit